.azure-pipelines/pull_request.yml

```diff
@@ -231,6 +247,15 @@ stages:
               source activate scipp-developer
               cmake --build . --target install --config Release
             displayName: 'Build Python library'
+          - bash: |
+              set -ex
+              cd package/scipp-package-test
+              source activate scipp-developer
+              mkdir -p build && cd build
+              cmake -DCMAKE_PREFIX_PATH="$(install_dir)" ..
+              cmake --build . --config Release
```

Comment:
You can also disable tbb at cmake time using the flag `disable_tbb` (need to check name). I'm not sure what is better.

Reply:
Then I would need to disable TBB when building *scipp*, which we don't want to.

Reply:
You are right we absolutely do not want to do that.